source/appModules/hxmail.py

```diff
@@ -0,0 +1,35 @@
+#A part of NonVisual Desktop Access (NVDA)
+#Copyright (C) 2016 NV Access Limited
+#This file is covered by the GNU General Public License.
+#See the file COPYING for more details.
+
+import appModuleHandler
+import UIAHandler
+from NVDAObjects.UIA.wordDocument import WordDocument
+
+class MailWordDocumentTreeInterceptor(WordDocument.treeInterceptorClass):
+
+	def _get_isAlive(self):
+		return super(MailWordDocumentTreeInterceptor,self).isAlive and self.rootNVDAObject.shouldCreateTreeInterceptor

```

Comment:
The document is always re-used, yes. Thus the only check we can really do is seeing if it goes from read-only to editable. You are right that therefore forcing a treeInterceptor on an editable document would not work. You would really need a different implementation of isAlive for forced treeInterceptos... but then again, it wouldn't die if you forced on an editable, then opened a read-only mail, then started a new editable amil.